addons/pos_mercado_pago/models/pos_payment_method.py

```diff
@@ -24,8 +24,16 @@ class PosPaymentMethod(models.Model):
         help="Enter your Point Smart terminal serial number written on the back of your terminal (after the S/N:)")
     mp_id_point_smart_complet = fields.Char()
 
-    def _get_payment_terminal_selection(self):
-        return super()._get_payment_terminal_selection() + [('mercado_pago', 'Mercado Pago')]
+    @api.onchange('use_payment_terminal')
+    def _onchange_use_payment_terminal(self):
+        super()._onchange_use_payment_terminal()
+        if self.use_payment_terminal == 'mercado_pago' and not self.mp_webhook_secret_key:
+            existing_payment_method = self.search([('use_payment_terminal', '=', 'mercado_pago'), ('mp_webhook_secret_key', '!=', False)], limit=1)
+            if existing_payment_method:
+                self.write({
```

Comment:
could you take this into account?

![image](https://github.com/user-attachments/assets/212a1f89-bfc9-4f63-852b-34ddd0d13339)

[reference](https://www.odoo.com/documentation/master/developer/reference/backend/orm.html#:~:text=will%20be%20ignored-,Danger,-Since)

Reply:
Hello @vlst-odoo 
Changes are done.